### PR TITLE
Resource group function test: cancel/terminate queries while they are

### PR DESCRIPTION
**Pull request by guofengrichard**
running or waiting in resource group.

### CODE REVIEW

File: src/test/isolation2/sql/resgroup/resgroup_func_concurrency.sql [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+-- end_ignore
+
+CREATE OR REPLACE VIEW rg_concurrency_view AS
+	SELECT waiting, waiting_reason, current_query, rsgname
+	FROM pg_stat_activity
+	WHERE rsgname='rg_concurrency_test';
+
+CREATE RESOURCE GROUP rg_concurrency_test WITH (concurrency=1, cpu_rate_limit=20, memory_limit=20);
+CREATE ROLE role_concurrency_test RESOURCE GROUP rg_concurrency_test;
+1:SET ROLE role_concurrency_test;
```
Comment (xiong-gang) [inline src/test/isolation2/sql/resgroup/resgroup_func_concurrency.sql]: what is the purpose to have 10 connection?

File: src/test/isolation2/sql/resgroup/resgroup_func_concurrency.sql [hunk 2]
```diff
@@ -0,0 +1,228 @@
+-- test1: cancel a query that is waiting for a slot
```
Comment (xiong-gang) [inline src/test/isolation2/sql/resgroup/resgroup_func_concurrency.sql]: i don't think the name of this file explains the purpose of the cases

**Comment by pivotal-ning-yu** [comment]
It's better to polish the commit message, current one's title is too long so it even can't be displayed completely on github.

I would suggest never use `git commit -m` to commit a message.

**Comment by guofengrichard** [comment]
Created a new PR 3310 for this commit. 
Close this one.

**Pull request closed without merging**